analytics/scala/src/main/scala/com/koverse/example/spark/WordCounter.scala

```diff
@@ -61,7 +61,10 @@ class WordCounter(
     import spark.implicits._
 
     // for each Record, tokenize the specified text field and count each occurrence
-    val wordCountDataset = inputRecordsDataset.flatMap(record => record.text.toLowerCase.split(tokenizationString))
+    inputRecordsDataset.show()
+    // Take the column that contains the text and tokenize and count the words
+//    val wordDF = inputDataFrame.explode(textFieldName, "word") { text: String => text.split(tokenizationString) }
```

Comment:
If this isn't needed, please remove it.

Reply:
I don't see it there. Maybe you were looking before my latest push?